pom.xml

```diff
@@ -189,11 +189,21 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.pivovarit</groupId>
+            <artifactId>parallel-collectors</artifactId>
+            <version>2.6.1</version>
+        </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>pipeline-utility-steps</artifactId>
             <scope>test</scope>
         </dependency>
+		<dependency>
```

Comment:
use spaces not tabs please to be consistent with the rest of the file

Reply:
My editors have no preference, so this keeps happening, gonna fix. I come from a place where using spaces makes the compilation 10% slower.